MVP/jenkins-jobs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+need_pip()
+{
+    echo "  We need 'pip' to install/update jenkins."
+    echo "  Please run 'easy_install pip' as root to install it."
+    exit 1
+}
+
+upgrade_jjb()
+{
+    which pip || need_pip
+    pip install --user --upgrade pip
+    pip install --user --upgrade jenkins-job-builder
```

Comment:
We shouldn't be doing this. jenkins-job-builder may break backward compatibility which is a risk. jenkins-job-builder==1.3.0 is tested to work.